tests/alembic_migrations/versions/v_84b7fc2596d5_add_utility_views/test_views.py

```diff
@@ -81,22 +88,14 @@ def test_obs_count_per_day_history(views_sesh):
         assert ocdh_count.count == obs_count.count
 
 
-def test_obs_with_flags(views_sesh):
-    # extensions = (
-    #     views_sesh.execute('''
-    #         SELECT * FROM pg_extension
-    #     ''')
-    # )
-    # print('### extensions', {
-    #     '{}: {}'.format(e.extname, e.extversion) for e in extensions
-    # })
```

Comment:
I'm guessing this will be reintroduced later on?

Reply:
No, I thought I had deleted it.

Reply:
Sorry, I just wasn't sure if the that test was no longer valid, or one of the tests that would come back after some changes further down the line.

Reply:
The _test_ is still there, but some cruft has been removed. If you look at the + side, you'll see that the function def is still there, just a little modified.

Reply:
I missed that, my mistake!